src/test/java/io/vertx/test/core/RedeploymentTest.java

```diff
@@ -88,7 +88,13 @@ public void setUp() throws Exception {
 
   @Override
   public void tearDown() throws Exception {
-    vertx.fileSystem().deleteRecursiveBlocking(tempDir.getPath(), true);
+    /* One of the temp files is a jar which is added to the classpath to test verticle redeploy on jar
+     * changes.  This jar remains locked by the JVM until all references to this class loader have been
+     * removed and garbage collected.  The following clean up statement fails because either there is
```

Comment:
Maybe use Junit TemporaryFolder like we do in other tests (e.g. HttpTest)? Perhaps this works better on Windows?